server/service/sessions.go

```diff
@@ -113,8 +112,9 @@ func (svc *Service) DeleteSession(ctx context.Context, id uint) error {
 ////////////////////////////////////////////////////////////////////////////////
 
 type loginRequest struct {
-	Email    string `json:"email"`
-	Password string `json:"password"`
+	Email                     string `json:"email"`
+	Password                  string `json:"password"`
+	SupportsEmailVerification bool   `json:"supports_email_verification"`
```

Comment:
Let's document the goal of this argument. (I'm always scared about client side arguments on login endpoints.)

Guessing this is set by UI and not set by API/fleetctl (to fail with a proper error on the latter)?

Reply:
Correct, and yep, I [have it documented](https://github.com/fleetdm/fleet/pull/24168/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64R62-R65).